{"name": "Негативный тест авторизации", "status": "passed", "steps": [{"name": "Шаг 1. Вводим ложные логин/пароль", "status": "passed", "start": 1738134570843, "stop": 1738134570861}], "parameters": [{"name": "browser_name", "value": "'chromium'"}, {"name": "username", "value": "'123'"}, {"name": "password", "value": "'admin'"}, {"name": "result", "value": "'Invalid credentials. Please try again.'"}], "start": 1738134565929, "stop": 1738134570861, "uuid": "51cc88a1-7f2e-420b-9dec-8dc4e0689b4d", "historyId": "784fbf7dbe03a7b6aabb771d2221e2ee", "testCaseId": "68772be64bb38f6a4c2b59109d7c35bc", "fullName": "test_login#test_login_failure", "labels": [{"name": "story", "value": "Login Feature"}, {"name": "severity", "value": "normal"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-TR3R9M3"}, {"name": "thread", "value": "45524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}